storybook/storybook-css/src/docs/typography.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Typografie" />
+
+# Typografie
+
+We gebruiken het lettertype Amsterdam Sans. De _fallback_ lettertypes zijn Arial en sans-serif.
```

Comment:
I personally like to include `system-ui`

Reply:
For what purpose? [This person](https://infinnie.github.io/blog/2017/systemui.html) doesn't seem to be a fan 😁